ace-setup-mq/readme.md

```diff
@@ -0,0 +1,28 @@
+# Instructions for setup-channel.sh
+## Descriptions 
+The script `setup-channel.sh` is used to create resources and set authority properly in QMGR for ACE sensor. 
```

Comment:
**thought**: The script name is not clear enough. It is not only just for set channel but also auth, listener, port etc. I'd propose something like `preconfig-ace-mon.sh` and `revert-preconfig-ace-mon.sh`?

Reply:
You are right, it is not for channel setup only. But I want to keep the name simple and easy to remember,  how about`prep-mq-for-ace.sh `and `revert-mq-for-ace.sh`?

Reply:
`mq-for-ace` is confusing as it is for ace mon not for ace iteself.

Reply:
`prep-mq` and `revert-mq` is short enough 😄